app/src/main/java/net/tensory/rxjavatalk/providers/BattleProvider.java

```diff
@@ -22,8 +30,6 @@ public BattleProvider(DragonManager dragonManager) {
      * @return hot Observable
      */
     public Observable<Battle> observeBattles() {
-        return Observable.merge(
-                northernFrontFeed.observeBattles(),
-                southernFrontFeed.observeBattles());
+        return battlesSubject;
```

Comment:
😹

Reply:
Great job figuring out it was the multiple subscriptions causing the problem!